README.md

```diff
@@ -36,10 +36,11 @@ To install the frontend please do the following:
 3. Install [ember-cli latest](https://www.npmjs.org/package/ember-cli): `npm install -g ember-cli@latest`.
    Depending on your [npm permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) you might need root access to install ember-cli.
 4. Install [bower](https://www.npmjs.org/package/bower): `npm install -g bower`
-5. Clone this repo with `git clone https://github.com/HospitalRun/hospitalrun-frontend`, go to the cloned folder and run `script/bootstrap`.
+5. Fork the master repository from `https://github.com/HospitalRun/hospitalrun-frontend` and clone the forked repository.
```

Comment:
I'm not sure why the change here.  For some folks cloning is all they will need to do.